app/models/customer.rb

```diff
@@ -26,14 +26,21 @@ class Customer < ApplicationRecord
   scope :ordered, -> { order('created_at DESC') }
   scope :with_name, ->(name) { where('name ilike ?', "%#{name}%") }
 
-
   # gem 'paranoia' doesn't run validation callbacks on restore
   before_restore :run_validations
 
   def to_s
     name.truncate(30)
   end
 
+  def indebtedness
+    result = { 'EUR': 0, 'USD': 0, 'RUB': 0 }
```

Comment:
@dissident not sure it's good to use predefined list of currencies. In future we will have a lot of currencies, so when we will add new currency, we will have to edit this code (and almost always will forget to do it). Instead I suggest to make as it is in current logic: have separate row for each customer-currency pair. In real case each customer will have only one currency.